sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java

```diff
@@ -672,14 +676,15 @@ private FacilityReferenceDto getLabReference(LabMessageDto labMessageDto) {
 			viaLimsCheckbox.setValue(Boolean.TRUE);
 			viaLimsCheckbox.setEnabled(false);
 		}
-
-		((ComboBox) sampleCreateComponent.getWrappedComponent().getField(PathogenTestDto.TEST_RESULT)).setValue(labMessageDto.getTestResult());
-		((ComboBox) sampleCreateComponent.getWrappedComponent().getField(PathogenTestDto.TEST_TYPE)).setValue(labMessageDto.getTestType());
+		// TODO currently just the first testReport is picked here. That must be temporary.
+		TestReportDto testReportDto = FacadeProvider.getTestReportFacade().getAllByLabMessage(labMessageDto.toReference()).get(0);
```

Comment:
When there is no test report, this creates

`Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
	at java.base/java.util.Objects.checkIndex(Objects.java:372)
	at java.base/java.util.ArrayList.get(ArrayList.java:458)
	at de.symeda.sormas.ui.labmessage.LabMessageController.getSampleCreateComponent(LabMessageController.java:680)
	at de.symeda.sormas.ui.labmessage.LabMessageController.createSample(LabMessageController.java:626)
	at de.symeda.sormas.ui.labmessage.LabMessageController.lambda$getCaseCreateComponent$17(LabMessageController.java:564)`

We should avoid that. It happened to me when I tried to process a lab message.